<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QMainWindow" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duadrotor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="leftFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QwtDial" name="leftMotorDial">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 255, 0);</string>
         </property>
         <property name="upperBound">
          <double>35.000000000000000</double>
         </property>
         <property name="scaleMaxMajor">
          <number>10</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="frameShadow">
          <enum>QwtDial::Sunken</enum>
         </property>
         <property name="mode">
          <enum>QwtDial::RotateNeedle</enum>
         </property>
         <property name="origin">
          <double>135.000000000000000</double>
         </property>
         <property name="maxScaleArc">
          <double>270.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QwtTextLabel" name="TextLabel">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="plainText">
          <string>Left Motor [4]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="balanceFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="balanceMeterPlace">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QwtTextLabel" name="balanceLbl">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="plainText">
          <string>Balance</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="rightFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QwtDial" name="rightMotorDial">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 255, 0);</string>
         </property>
         <property name="upperBound">
          <double>35.000000000000000</double>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="origin">
          <double>135.000000000000000</double>
         </property>
         <property name="maxScaleArc">
          <double>270.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QwtTextLabel" name="rightLbl">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="plainText">
          <string>Right Motor [2]</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
